src/components/elements/table/GenericTable.tsx

```diff
@@ -176,10 +188,48 @@ export const renderHeaderCellContents = <T extends { id: string }>(
   );
 };
 
+export const renderLinkedCellContents = <T extends { id: string }>(
+  rowLink: To,
+  row: T,
+  render: ColumnDef<T>['render'],
+  index: number,
+  rowLinkState: LocationState | undefined = undefined
+) => {
+  return (
+    <RouterLink
+      to={rowLink}
+      state={rowLinkState}
+      plain
+      sx={{
+        height: '100%',
+        verticalAlign: 'middle',
+        display: 'block',
+        '&.Mui-focusVisible': {
+          outlineOffset: '-2px',
+        },
+      }}
+      tabIndex={index === 0 ? 0 : -1}
```

Comment:
This could use a comment. My guess is: unless this is the first cell in the row (aka primary column) make the link NOT tabbable by passing `-1`. I think `0` might as well be undefined because links are tabbable by default.